framework/shortcuts/internal/shortcutscontroller.cpp

```
@@ -34,6 +38,26 @@
 using namespace muse::shortcuts;
 using namespace muse::actions;
 using namespace muse::rcommand;
+using namespace muse::ui;
+
+static std::map<std::string, Command> compatActionToCommand = {
```

Clang (14.0.6)

[warning] variable 'compatActionToCommand' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)